src/main/scala/org/broadinstitute/dsde/rawls/user/UserService.scala

```diff
@@ -747,8 +778,9 @@ class UserService(protected val userInfo: UserInfo, val dataSource: SlickDataSou
           dataAccess.rawlsBillingProjectQuery.load(projectName) flatMap {
             case None =>
               for {
-                _ <- DBIO.from(gcsDAO.createProject(projectName, billingAccount, billingProjectTemplate))
-                _ <- createBillingProjectInternal(dataAccess, projectName, Set(RawlsUser(userInfo)), Set.empty, CreationStatuses.Creating)
+                groups <- createBillingProjectGroups(dataAccess, projectName, Set(RawlsUser(userInfo)))
+                _ <- DBIO.from(gcsDAO.createProject(projectName, billingAccount, billingProjectTemplate, groups))
```

Comment:
this is also a much greater refactor... createBillingProjectGroups also makes google calls and I would like create project to do less and do the rest asynchronously.